### PR TITLE
update contacts app and companies app to v3

### PR DESCRIPTION
**Pull request by ksvirkou-hubspot**
update to v3 + code clean

### CODE REVIEW

**Comment by atanasiuk-hubspot** [comment]
Could you please describe what the bug is and how this PR helps? Thanks

**Comment by ksvirkou-hubspot** [comment]
> Could you please describe what the bug is and how this PR helps? Thanks

this pr update companies app and contacts app to hubspot to 3.0.1

File: sample-apps/contacts-app/src/views/properties/show.php [hunk 1]
```diff
@@ -41,9 +41,9 @@
         <input type="text" name="group_name" id="group-name" value="<?php echo htmlentities($property->getGroupName()); ?>" />
 
         <label for="type">Type</label>
-        <input readonly type="text" name="type" id="type" value="<?php echo htmlentities($property->getType()); ?>"<?php echo $readOnly; ?> />
```
Comment (atanasiuk-hubspot) [inline sample-apps/contacts-app/src/views/properties/show.php:44]: Is this change intended?

Comment (ksvirkou-hubspot) [reply sample-apps/contacts-app/src/views/properties/show.php:44]: It is just code clean

**Review by atanasiuk-hubspot — APPROVED** [review]
